website/docs/basics/autowiring.md

```diff
@@ -0,0 +1,218 @@
+---
+id: autowiring
+title: Autowiring
+sidebar_label: Autowiring
+---
+
+[![docs-source](https://img.shields.io/badge/source-eigthshift--libs-blue?style=for-the-badge&logo=php&labelColor=2a2a2a)](https://github.com/infinum/eightshift-libs)
+
+What is autowiring?
+
+*Autowiring is an exotic word that represents something very simple: the container's ability to create and inject dependencies automatically.*
+
+The quote was used from [PHP-DI](https://php-di.org/doc/autowiring.html) library that we use.
+
+In a nutshell, we use a dependency injection container to load all your service classes. With DI container, we also use the PSR-4 coding standard that allows us to predict how your file/folder structure will look like in correlation to the namespace. With that in mind, you don't have to worry about when your class is required, and when it loads, our autowiring takes care of everything. You need to follow a few coding standards.
```

Comment:
```suggestion
In a nutshell, we use a dependency injection container to load all your service classes. Alongside the DI container, we are also using the PSR-4 coding standard for autoloading. That allows us to predict how your file/folder structure will look like in correlation to the namespace. With that in mind, you don't have to worry about requiring your classes. Our autoloading takes care of everything. All you need to do is follow a few coding standards so that you can automatically inject other classes as dependencies.
```